tuscan/wrapper_gen.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python2
+#
+# Copyright 2016 Kareem Khazem. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Generation of compiler wrappers and Dockerfile for a toolchain.
+
+
+from schemata import binutil_schema
+
+import argparse
+import jinja2
+import logging
+import os.path
+import subprocess
+import tempfile
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("toolchain")
+    args = parser.parse_args()
+
+    logging.basicConfig(format="%(asctime)s %(message)s")
+
+    make_package_dir = os.path.join("toolchains", "make_package",
+            args.toolchain)
+
+    required_files = [
+        os.path.join(make_package_dir, "binutil_transforms.yaml"),
+        os.path.join(make_package_dir, "Dockerfile"),
+        os.path.join("tuscan", "compiler_wrapper.c"),
+    ]
+    for f in required_files:
+        if not os.path.exists(f):
+            logging.error("File not found: '%s'" % f)
+            exit(1)
+
+    with open(os.path.join(make_package_dir, "binutil_transforms.yaml")) as f:
```

Comment:
I'd consider using a different name for the file, `binutil` is confusing since it can refer to GNU Binutils (that's what I originally thought).